reference-data/holdings-sources/Marc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "id": "036ee84a-6afd-4c3c-9ad3-4a12ab875f59",
+  "name": "MARC",
+  "source": "marc"
```

Comment:
Scenarios 6 and 7 of [UIIN-1229](https://issues.folio.org/browse/UIIN-1229) state that the two options for this property are `folio` and `local`.

Why does this record have a different `source` value to those?

Reply:
I've never heard of UIIN-1229.

I added the reference data because I thought you asked me to:

"As I understand it, the two sources: FOLIO and MARC are intended to be provided as reference data, please could those be added."

Reply:
I understand that it is confusing and that maybe I should have consolidated the requirements from the two front-end stories before the back end work started. 

[MODINVSTOR-542](https://issues.folio.org/browse/MODINVSTOR-542) is intended to provide API changes that meet the needs of [UIIN-1123](https://issues.folio.org/browse/UIIN-1123), [UIIN-1229](https://issues.folio.org/browse/UIIN-1229) and [UIIN-1134](https://issues.folio.org/browse/UIIN-1134)

As I understand it, the holdings sources that are needed are `FOLIO` and `MARC`. I believe both of those should have `source` properties of `folio` and be uneditable, whereas holdings sources created by the UI should have a source property of `local` and be editable.

Does that make sense?

Would it be helpful if I consolidated the requirements from these UI stories into the backend story? I don't tend to do that because it leads to a lot of duplication of requirements.

Reply:
I think I just came onto this issue late in its life cycle, so I don't have the best frame of reference.

Reply:
This isn't the first time this kind of issue has been confusing. There is a mismatch between the front end and back end work, and typically the requirements are very heavily biased towards the front end.

It hasn't come up in a while, so I'd likely forgotten how painful it can be. I've added a topic to the teams retrospective for folks to reflect upon it.

Reply:
Also, please wait before actioning my comments about whether holdings sources should be editable. I need to get those requirements clarified.